src/prefect/_experimental/bundles.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import annotations
+
+import asyncio
+import base64
+import gzip
+import logging
+import multiprocessing
+import multiprocessing.context
+import os
+from typing import Any, TypedDict
+
+import cloudpickle
+
+from prefect.client.schemas.objects import FlowRun
+from prefect.context import SettingsContext, get_settings_context, serialize_context
+from prefect.exceptions import Abort, Pause
+from prefect.flow_engine import run_flow
+from prefect.flows import Flow
+from prefect.settings.context import get_current_settings
+from prefect.settings.models.root import Settings
+
+
+class Bundle(TypedDict):
+    serialized_function: str
+    serialized_parameters: str
+    serialized_context: str
+    serialized_flow_run: dict[str, Any]
```

Comment:
considering all of these attributes have `serialized_` it seems the object will be less verbose if we either remove that prefix entirely or call it `SerializedBundle` or something like that